frontend/src/features/mapSideBar/containers/MapSideBarContainer.tsx

```diff
@@ -215,7 +213,7 @@ const MapSideBarContainer: React.FunctionComponent<IMapSideBarContainerProps> =
           `${nameSpace}longitude`,
           leafletMouseEvent?.latlng.lng || 0,
         );
-        droppedMarkerSearch(movingPinNameSpace, leafletMouseEvent?.latlng);
+        !parcelId && droppedMarkerSearch(movingPinNameSpace, leafletMouseEvent?.latlng);
```

Comment:
searching the parcel layer for the dropped marker location isn't necessary if this is an update.